### ISSUE TITLE
Zip code data missing from original list after campaign survey export

### ISSUE THREAD

**Issue by deasterdaywfp** [WorkingFamilies/Spoke_legacy#84]
upon export of campaign survey data, zip code field returns blank for every single contact.

**Comment by deasterdaywfp** [comment]
Work around found. Closing

**Comment by joemcl** [comment]
Was the zipcode not loaded with the list, maybe? If so what is the workaround?

**Comment by deasterdaywfp** [comment]
No, this is missing upon export only. It was and always is included in
lists I upload.

*Dylan Easterday*
*National Volunteer & Distributed Organizing Manager - Working Families
Party*
WFP4theMany.org
*Text *(347) 460-9451 or Schedule https://calendly.com/deasterday
Email deasterday@workingfamilies.org
Sign up to volunteer here <http://www.wfp4themany.org>
